pkgs/by-name/fi/fish/package.nix

```diff
@@ -292,6 +292,9 @@ stdenv.mkDerivation (finalAttrs: {
     ]
     ++ lib.optionals stdenv.hostPlatform.isDarwin [
       "-DMAC_CODESIGN_ID=OFF"
+    ]
+    ++ lib.optionals (stdenv.buildPlatform != stdenv.hostPlatform) [
+      "-DRust_CARGO_TARGET=${stdenv.hostPlatform.rust.rustcTarget}"
```

Comment:
We can set this unconditionally.

Reply:
I mainly did not to avoid a rebuild in the base case

Reply:
We can batch it with the 4.0.1 bump in #389486, since we'll be rebuilding anyway.

But in any case, it's not a big deal in this case.

Reply:
Yeah, sounds good, I'll do that.